inhouse_bot/cogs/admin_cog.py

```diff
@@ -194,22 +193,21 @@ async def verify(self, ctx: commands.Context, user: discord.Member, *name_arg: s
         name = ' '.join(name_arg)
         server_id = ctx.guild.id
         try:
-            summoner = self.lol_Api.get_summoner_by_name(name)
-        except HTTPError as ex:
-            if ex.response.status_code == 404:
+            summoner = await get_summoner_by_name(name)
+        except PyotException as ex:
+            if ex.code == 404:
                 return await ctx.send(f"Couldn't find Summoner name: {name}")
             # anything else gets raised
             raise ex
 
-        puuid = summoner["puuid"]
-        summoner_name = summoner["name"]
+        summoner_name = summoner.name
 
```

Comment:
Better just to make var ```summoner_puuid``` just to follow the flow of the other code.

Reply:
good catch